app.js

```diff
@@ -24,143 +24,160 @@ SurveyItem.prototype.buildSurveyItem = function(conatainerId){
   newSurveyItemDesc.setAttribute('class','surveyItemDesc');
   newImage.setAttribute('src',this.itemImageFile);
   newImage.setAttribute('alt',this.itemName);
+  newImage.setAttribute('name',this.itemName);
   newItemName.innerText = this.itemName;
   newItemDesc.innerText = this.itemDesc;
   container.appendChild(newSurveyImageBox);
   container.appendChild(newSurveyItemDesc);
   newSurveyImageBox.appendChild(newImage);
   newSurveyItemDesc.appendChild(newItemName);
   newSurveyItemDesc.appendChild(newItemDesc);
+  this.setShownValues();
 };
 
 SurveyItem.prototype.buildRandomChartValues = function() {
   this.numberOfClicks = Math.floor(Math.random() * 10);
   this.numberOfTimesShown = Math.floor(Math.random() * 15 + 10);
 };
 
-var boots = new SurveyItem('Jabba','$19.50','Ubb maul mace gev jerjerrod dressellian. Falleen arkanis iridonian desann skywalker greedo priapulin hapes.','images/boots.jpg');
-var chair = new SurveyItem('Leah','$99.99','Neimoidia bollux momaw droid saffa sing b4-d4 tierce. Saurin roan jarael kastolar nunb ralter.','images/chair.jpg');
-var scissors = new SurveyItem('Han','$7.89','Shistavanen phlog depa oppo qui-gon t88 dorvalla. Ailyn darth irek max jabba finis kiffar antonio bith.','images/scissors.jpg');
-var water_can = new SurveyItem('Boba','$10.00','Quence mirta kalee qui-gon halla seerdon bibble. Calamari jacen tund tierce hutt salacious alderaan.','images/water_can.jpg');
-var wine_glass = new SurveyItem('Vader','$23.99','Vader saffa arkanis kyle givin yavin. Utapaun selkath dat mccool zuggs medon derlin kathol secura.','images/wine_glass.jpg');
-var boots2 = new SurveyItem('Jabba2','$19.50','Ubb maul mace gev jerjerrod dressellian. Falleen arkanis iridonian desann skywalker greedo priapulin hapes.','images/boots.jpg');
-var chair2 = new SurveyItem('Leah2','$99.99','Neimoidia bollux momaw droid saffa sing b4-d4 tierce. Saurin roan jarael kastolar nunb ralter.','images/chair.jpg');
-var scissors2 = new SurveyItem('Han2','$7.89','Shistavanen phlog depa oppo qui-gon t88 dorvalla. Ailyn darth irek max jabba finis kiffar antonio bith.','images/scissors.jpg');
-var water_can2 = new SurveyItem('Boba2','$10.00','Quence mirta kalee qui-gon halla seerdon bibble. Calamari jacen tund tierce hutt salacious alderaan.','images/water_can.jpg');
+SurveyItem.prototype.setShownValues = function () {
```

Comment:
Good use of a method here. Would have been easy to put this logic directly in `buildSurveyItem`.